src/libs/Network/index.js

```diff
@@ -40,8 +40,8 @@ function post(command, data = {}, type = CONST.NETWORK.METHOD.POST, shouldUseSec
             appversion: pkg.version,
         };
 
-        const shouldPersist = lodashGet(request, 'data.persist', false);
-        if (shouldPersist) {
+        const shouldRetry = lodashGet(request, 'data.retry', false);
+        if (shouldRetry) {
```

Comment:
Idk... I think we don't really need a `retry: true` at all? 

What is passing `persist` now besides tests? I think we can just remove this entirely?

Reply:
Yeah, just tests... great will do!